modules/exploits/multi/http/motioneye_auth_rce_cve_2025_60787.rb

```diff
@@ -0,0 +1,404 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Remote Code Execution Vulnerability in MotionEye (CVE-2025-60787)',
+        'Description' => %q{
+          MotionEye is vulnerable to OS Command Injection in configuration parameters such as image_file_name.
+          Unsanitized user input is written to Motion configuration files,
+          allowing remote authenticated attackers with admin access to achieve code execution when Motion is restarted.
```

Comment:
Hmm, you say when restarted but this isn't noted in the documentation file. If the server requires restarting for this vulnerability to trigger we should note this somewhere. 

From the exploitation output you show in your documentation I don't think this exploit requires the Motion server to be restarted so I think this line is incorrect. It appears to require one to add a new camera and then remove it, and the removing of the camera triggers the vulnerability.

That being said, please feel free to correct me if I am wrong about this.